src/odin/directionsbuilder.cc

```diff
@@ -101,8 +101,6 @@ void DirectionsBuilder::Build(Api& api) {
 
       // Return trip directions
       PopulateDirectionsLeg(options, &etp, maneuvers, trip_directions);
-
-      LOG_INFO("maneuver_count::" + std::to_string(trip_directions.maneuver_size()));
```

Comment:
Why were these removed? Is this because of the excessive logging we've seen recently? Shouldn't that be fixed by changing the log level, instead of removing the log statement?

Reply:
not needed - from old debug sessions